appointment-svc/models/decryption.ts

```diff
@@ -0,0 +1,25 @@
+import crypto, { CipherGCMTypes } from 'crypto'
+
+const secret_key = process.env.HASH_SECRET_KEY
```

Comment:
What are CipherGCMTypes?

Reply:
An internal type from crypto library. TS wanted it for the encryption method variable, even though it is just a string, but apparently there are some possible complicated use cases this type is used for. I am not aware of those